src/nodemetadata.cpp

```diff
@@ -77,11 +90,38 @@ bool NodeMetadata::empty() const
 	return Metadata::empty() && m_inventory->getLists().size() == 0;
 }
 
+bool NodeMetadata::getPrivate(const std::string &name) const
```

Comment:
The function sounds wrong. Should be `isPrivate`, as it checks for the existence of an entry in the private list.